python/ray/data/tests/test_parquet.py

```diff
@@ -181,6 +181,11 @@ def test_parquet_read_basic(ray_start_regular_shared, fs, data_path):
     assert sorted(values) == [1, 2, 3, 4, 5, 6]
     assert ds.schema().names == ["one"]
 
+    # Test concurrency.
+    ds = ray.data.read_parquet(data_path, filesystem=fs, concurrency=1)
+    values = [s["one"] for s in ds.take()]
+    assert sorted(values) == [1, 2, 3, 4, 5, 6]
```

Comment:
`read_parquet` could launch more than one concurrent read task and this assertion would still pass. Is there any easy way to test that the concurrency is enforced?

Reply:
Yes, planning to add some test similar to `est_backpressure_policies.py:test_basic`. But good callout.